CONTRIBUTING.md

```diff
@@ -45,11 +45,12 @@ You'll want to add your change in the `## [Upcoming]` section, since it will mos
 
 Finally, remember that Changelogs are for humans, not machines. You should be describing what changed, from a user's perspective, not how it was changed from a technical perspective (that is what commit messages are for).  
 
-[issues list]: /ImpactDevelopment/Installer/issues
-[new issue]: /ImpactDevelopment/Installer/issues/new
-[fork]: /ImpactDevelopment/Installer/fork
-[devenv]: /ImpactDevelopment/Installer#setting-up-a-development-environment
+[issues list]: /../../issues
+[new issue]: /../../issues/new
+[fork]: /../../fork
```

Comment:
TBH these are probably better off as absolute links since this file could be read from a cloned repo rather than on GitHub.